noWord/common/NWGenerator.py

```diff
@@ -84,6 +85,14 @@ def __init__(self, aSourcePath, aOutputPath, extPluginFolders=[], dumpContent=Fa
             if hasattr(NoWordDecoration, deco):
                 self.doc.addDecoration(getattr(NoWordDecoration, deco), deco in enabledDecorations)
 
+        enabledTransformations = self.context.docInfo["transformations"] if "transformations" in self.context.docInfo else []
+        additionalTransformations = self.context.docInfo["additionalTransformations"] if "additionalTransformations" in self.context.docInfo else []
```

Comment:
What is the difference between enabledTransformations and additionalTransformations ?

Reply:
You are right. Unnecessary complexity. I removed it.
DONE